docs/get_started.rst

```diff
@@ -1,17 +1,388 @@
+
+Inside your terminal application, enter the following command::
+
+    pipx install cumulusci
+
+Once finished `verify your installation`_.
+
+
+
+Verify Your Installation
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+In a new terminal window, verify that CumulusCI installed correctly by running ``cci version``.
+
+.. code-block:: console
+
+    $ cci version
+    CumulusCI version: 3.20.0 (/path/to/bin/cci)
+    Python version: 3.8.5 (/path/to/bin/python)
+
+    You have the latest version of CumulusCI.
+
+You can also use this command to check whether your CumulusCI installation is up to date.
+
+Still need help? Feel free to submit a question on our `Trailblazer community group <https://trailblazers.salesforce.com/_ui/core/chatter/groups/GroupProfilePage?g=0F9300000009M9ZCAU>`_.
+
+
+
+Setup SFDX 
+----------
+
+While it's possible to use ``cci org connect <org_name>`` to connect to a persistent org, we recommend working with scratch orgs created by Salesforce DX.
+
+If you haven't set up Salesforce DX, follow these steps:
+
+1. `Install Salesforce CLI <https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_setup_install_cli.htm>`_
+2. `Enable Dev Hub Features in Your Org <https://developer.salesforce.com/docs/atlas.en-us.228.0.sfdx_dev.meta/sfdx_dev/sfdx_setup_enable_devhub.htm>`_
+3. `Connect SFDX to Your Dev Hub Org <https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_auth_web_flow.htm>`_ 
+        .. important:: Be sure to use the ``--setdefaultdevhubusername`` option!
+
+If you have the ``sfdx`` command installed, are connected to your devhub, and set the ``defaultdevhubusername`` config setting (use the ``sfdx force:config:list`` command to verify!), you're now ready to use ``cci`` with ``sfdx``.
+
+SFDX supports multiple DevHubs, so CumulusCI uses the one set as ``defaultdevhubusername`` when creating scratch orgs.
+
+Learn more about Salesforce DX at https://developer.salesforce.com/platform/dx.
+
+
+
 Connect to GitHub
 -----------------
 
-Work on an existing CumulusCI project
+In order to allow CumlusCI to work with your CumulusCI projects in GitHub, connect GitHub as a service in ``cci``.
+
+First, `create a new personal access token <https://github.com/settings/tokens/new>`_ with both "repo" and "gist" scopes specified. (Scopes appear as checkboxes when creating the personal access token in GitHub.) Copy the access token to use as the password when configuring the GitHub service.
+
+[ EXAMPLE PHOTO OF ACCESS TOKEN TO COPY??? ]
+
+Next, run the following command and provide your GitHub username and access token as the password::
+
+    $ cci service connect github
+
+Verify the GitHub service is connected by running ``cci service list``:
+
+.. image:: images/service-list.png
+
+Once you've configured the ``github`` service it will be available to **all** CumulusCI projects.
+
+Services are stored in the global CumulusCI keychain by default.
+
+
+
+Work on an Existing CumulusCI Project
```

Comment:
```suggestion
Work On an Existing CumulusCI Project
```
I believe this is a verb phrase, so we do capitalize the "O": https://confluence.internal.salesforce.com/display/DOCTEAM/title+capitalization